libraries/c_sdk/standard/mqtt_v2/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+afr_module()
+
+afr_set_lib_metadata(ID "mqttv2")
+afr_set_lib_metadata(DESCRIPTION "This library implements the MQTT v2 protocol that enables \
+communication with AWS IoT. MQTT is an ISO standard publish-subscribe-based messaging protocol.")
+afr_set_lib_metadata(DISPLAY_NAME "MQTTv2")
+afr_set_lib_metadata(CATEGORY "Connectivity")
+afr_set_lib_metadata(VERSION "1.0.0")
+afr_set_lib_metadata(IS_VISIBLE "true")
+
+set(src_dir "${CMAKE_CURRENT_LIST_DIR}/src")
+set(inc_dir "${CMAKE_CURRENT_LIST_DIR}/include")
+set(test_dir "${CMAKE_CURRENT_LIST_DIR}/utest")
+
+# Compile BLE MQTT serializers on supported platforms.
+if(${BLE_SUPPORTED})
+    set(extra_mqtt_sources "${src_dir}/iot_ble_mqtt_serialize.c")
+    set(extra_mqtt_dependencies AFR::serializer AFR::ble)
+endif()
+
+# Enable test access and mocks if building tests.
+if(${AFR_IS_TESTING})
+    list(APPEND extra_mqtt_test_includes "${test_dir}/access")
+    list(APPEND extra_mqtt_test_includes "${test_dir}/mock")
+endif()
+
+afr_module_sources(
+    ${AFR_CURRENT_MODULE}
+    PRIVATE
+        "${src_dir}/mqtt.c"
+        "${src_dir}/mqtt_lightweight.c"
+        "${src_dir}/mqtt_state.c"
```

Comment:
Should be able to use `${MQTT_SOURCES}` by including `mqttFilePaths.cmake`. Same with `${MQTT_INCLUDE_PUBLIC_DIRS}` and `${MQTT_INCLUDE_PRIVATE_DIRS}`.

Reply:
Resolving this because we would need to adjust mqttFilePaths.cmake to work with paths in AFR. Kinda defeats the purpose, but this should change with the tree sifting tool.

Reply:
Yea. That's correct. we need to use the paths from filepaths.cmake when we move to tree stitching.